libs/hdf-converters/src/prisma-mapper.ts

```diff
@@ -0,0 +1,181 @@
+import {ExecJSON} from 'inspecjs';
+import _ from 'lodash';
+import {version as HeimdallToolsVersion} from '../package.json';
+import {
+  BaseConverter,
+  ILookupPath,
+  MappedTransform,
+  parseCsv
+} from './base-converter';
+
+export type PrismaControls = {
+  records: Record<string, string>[];
+};
+// There is a Common Vulnerability Scoring System (cvss) field that contains a cvss qualitative severity rating,
+// however this field does not correlate to the "Severity" (low, moderate, important, high, and critical) field
+// that is mapped to the hdf "impact" field. Severity is mapped to the SEVERITY_LOOKUP"
+const SEVERITY_LOOKUP: Record<string, number> = {
+  low: 0.3,
+  moderate: 0.5,
+  important: 0.7,
+  high: 0.9,
+  critial: 1
+};
+
+function getTitle(item: Record<string, string>): string {
+  const hostName = _.get(item, 'Hostname');
+  const distro = _.get(item, 'Distro');
+  const type = _.get(item, 'Type');
+  return hostName.toString().concat(distro, type);
+}
+
+export class PrismaControlMapper extends BaseConverter {
+  data: PrismaControls = {
+    records: []
+  };
+  mappings: MappedTransform<ExecJSON.Execution, ILookupPath> = {
+    platform: {
+      name: 'Heimdall Tools',
+      release: HeimdallToolsVersion,
+      target_id: 'Prisma Cloud Scan Report'
+    },
+    version: HeimdallToolsVersion,
+    statistics: {
+      duration: null
+    },
+    profiles: [
+      {
+        name: 'Palo Alto Prisma Cloud Tool',
+        version: HeimdallToolsVersion,
```

Comment:
this should be specifying the version of the profile/test/prisma/whatever, not heimdall tools.  it also should not be "package version".  if that information is not provided, leave it as the empty string.